site/en/blog/user-agent-reduction-deprecation-trial/index.md

```diff
@@ -0,0 +1,99 @@
+
+## How do I participate in the User-Agent Reduction deprecation  trial?
+
+### Register for the trial
+
+To register for the origin trial and get a token for your domains, visit the Trial for [User Agent Reduction Deprecation page](/origintrials/#/view_trial/2608710084154359809).
+
+### Setup
+
+Once you've registerd for the trial, update your HTTP response headers with the following:
+
+1.  Add `Origin-Trial: <ORIGIN TRIAL TOKEN>` to your HTTP response header. <`ORIGIN TRIAL TOKEN`> contains the token you got when registering for the origin trial.
+1.  Add `Accept-CH: Sec-CH-UA-Full` to your HTTP response header. Setting `Accept-CH` will only cause the reduced User-Agent string to be sent on subsequent requests to the origin. 
+1. To resend the first navigation request with the reduced User-Agent string, add `Critical-CH: Sec-CH-UA-Full` to your HTTP response header, in addition to the `Accept-CH` and `Origin-Trial` headers.
+1.  If you want third-party subresource requests to also receive the reduced UA string, you have two options:
+    - Add a `Permissions-Policy` header with the third-party domains that should receive the reduced UA.
+        -  To allow a named list of third-party domains, add `Permissions-Policy: ch-ua-full=(self "[https://google.com](https://google.com)")`.
+        -  To allow all third-party domains, add `Permissions-Policy: ch-ua-full=*`.
+    - Add an `Accept-CH` meta tag with the third-party domains that should receive the reduced UA (only in Chrome 100 and above).
+        -  To allow a named list of third-party domains, add `<meta name="accept-ch" content="ch-ua-full=( https://google.com )">`.
+        -  It's not possible to delegate to all third-party domains via `*` in the meta tag.
+1. Load your website in Chrome 100 (or later) and continue receiving the full User-Agent string.
+
+### Demo
+
+See [https://uard-ot-demo.glitch.me](https://uard-ot-demo.glitch.me) for a demonstration of the trial (along with the source code).
+
+## How do I validate that the trial is working?
+
+To validate that the origin trial is working, examine the request headers and ensure the following:
+
+1.  The User-Agent header contains the full version. It shouldn't contain any of the reduced values (found in the [list of samples of reduced User-Agent strings](https://www.chromium.org/updates/ua-reduction#TOC-Sample-UA-Strings:-Phase-4)). An easy way to tell is that the Chrome minor version string should **not** be `0.0.0`.
+1.  The `Sec-CH-UA-Full` header is set to `?1`.
+
+The initial response's headers containing the origin-trial token should look like:
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/AIlPxFBDvO7jQrXuKfWU.png", alt="", width="800", height="175" %}
```

Comment:
I don't want to block this post from going live, but perhaps post-launch it would be a good idea to either switch this to fenced code block or add meaningful `alt` text? The fenced code block would probably be the most accessible, since the exact content of the image could be reproduced cleanly as text, and you could omit the headers that are not relevant to the example at hand.

```text
accept-ch: ...
origin-trial: ...
```

Reply:
Excellent suggestion, I'll add the code block in a follow up PR.